backend-tests/run

```diff
@@ -222,6 +226,10 @@ for suite in "${TEST_SUITES[@]}"; do
             ;;
     esac
 
+    if [ -n "$K8S" ]; then
+        COMPOSE_CMD="${COMPOSE_CMD_BASE}"
```

Comment:
I run the tests with the following command:
```
$ time K8S=1 GATEWAY_HOSTNAME=staging.hosted.mender.io STRIPE_API_KEY=rk_test_Nc3enGgRMq1y0jE90CfxmqYv00vDwGylov ./run --no-download -s enterprise -vvv
```

I explored the possibility to run the "open source" tests, but it makes no sense because the open source tests are not "open source" in the terms of plans, but single tenant. You were running open source tests using the enterprise components, for example, and it works fine but you have to start the components in a single tenant deployment. In the staging environment this is not possible, obviously, so I think it is better to stick to the enterprise filter (which, in reality, means multi-tenant).

I can hard-code the filter here if you think it is better, but I still want to respect the "-k" from command line if provided.

Reply:
I understood this wrong, sorry. `COMPOSE_CMD_BASE` is basically the test runner, nothing to do with OS/Enterprise docker-compose files (!). And it is up to the caller to pass the suite to run. Fine.

Yes, I agree in that doesn't make sense to run the OS tests. If any, we could check in the script if `K8S` is set, only enterprise is allowed (i.e. check `TEST_SUITES=( "enterprise" )`). But this is not important neither, we know how to run them.